client/CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ All notable changes to this project will be documented in this file.  The format
 * Change `put-deploy` and `make-deploy` subcommands to support transfers.
 * Change `make-deploy`, `make-transfer` and `sign-deploy` to not overwrite files unless `--force` is passed.
 * Change `make-deploy`, `make-transfer` and `sign-deploy` to use transactional file writing for enhanced safety and reliability.
-
+* Add `keygen::generate_files` to FFI.
```

Comment:
This should be part of the `### Added` section above.

Reply:
💯